relayer/relayer/src/handlers/azero.rs

```diff
@@ -183,6 +194,22 @@ impl AlephZeroEventHandler {
     }
 }
 
+async fn past_request(
+    most: &Most<SignedEthConnection>,
+    committee_id: U256,
+    address: Address,
+) -> Result<bool, AlephZeroEventHandlerError> {
+    if most.is_in_committee(committee_id, address).await? {
+        return Ok(false);
+    }
+
+    if committee_id > most.committee_id().await? {
+        error!("Request from a future committee {committee_id} - this likely indicates MOST contracts misconfiguration");
```

Comment:
if you;re not crashing log level should be  <= `warn`

Reply:
To me this also seems like a case where you can return an error and crash/(trigger circuit breaker). It is certainly unexpected that you see a request from a committee that you don't know. This would mean there is a problem with the bridge, or you are not synced.